std_msgs/Header header
int32 board_id
string board_info
